ui/Screen/Profile/Projects.svelte

```diff
@@ -80,13 +90,30 @@
       style="margin: 0 auto;">
       <Flex
         style="flex: 1; padding: 24px 16px 24px 24px;"
-        dataCy={`project-list-entry-${project.metadata.name}`}>
+        dataCy={`project-list-entry-${project.id}`}
+        on:mouseenter={project.stats ? null : handleMouseenter}
+        on:mouseleave={project.stats ? null : handleMouseleave}>
         <div slot="left">
-          <ProjectCard {...projectCardProps(project)} />
+          {#if project.metadata}
+            <ProjectCard {...projectCardProps(project)} />
+          {:else}
+            <ProjectCard
+              title="Unreplicated project"
+              description={project.id} />
+          {/if}
         </div>
 
         <div slot="right" style="display: flex; align-items: center;">
-          <Stats stats={statsProps(project.stats)} />
+          {#if project.stats}
```

Comment:
I have a PR up for integrating the actual project stats; let's revisit this & pair maybe once it's merged